src/components/projects/Project.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="projects-settings-page">
+    <div class="settings-window-page__title">
+      {{ $t('projects.settings.project.title') }}
+    </div>
+    <form
+      v-if="project"
+      @submit.prevent="save"
+    >
+      <div class="project-settings__inline-elements">
```

Comment:
better create `fieldset` with two inputs pulled to the left. And the picture component at the right